src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlServerVirtualNetworkRule.md

```diff
@@ -83,42 +98,13 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Confirm
```

Comment:
@markcowl It looks like this cmdlet does not use shouldprocess so it shouldn't have Confirm or WhatIf.  I don't know if it was generated incorrectly in the first place or if the cmdlet has changed but I believe this change is correct.